### ISSUE TITLE
[Bug Report] No focus on a v-chip in a v-chip-group

### ISSUE THREAD

**Issue by cotedav** [vuetifyjs/vuetify#10513]
### Environment
**Vuetify Version:** 2.2.8
**Vue Version:** 2.6.10
**Browsers:** Chrome 79.0.3945.130
**OS:** Windows 10

### Steps to reproduce
Try to focus the chips: we can see that in the tab order, the element get the focus, but no visual hint is present. The "Enter" key doesn't trigger the chip selection either. If we continue tabbing until the focus reaches the button, we can see that the focus actually works as intended on this component.

### Expected Behavior
- A visual focus on the chip, like in the Usage example here: https://vuetifyjs.com/en/components/chip-groups#chip-groups.

- Enter should activate the chip

### Actual Behavior
- No focus on the chip

- Enter key does not activate the chip

### Reproduction Link
<a href="https://codepen.io/cotedav/pen/wvaBzQE?editable=true&editors=101" target="_blank">https://codepen.io/cotedav/pen/wvaBzQE?editable=true&editors=101</a>

**Comment by lioneaglesolutions** [comment]
### Stale Issue

This issue is being closed due to inactivity.

If you have any additional questions, please reach out to us in our [[Discord community](https://community.vuetifyjs.com/)](https://community.vuetifyjs.com/).
